Dockerfile

```diff
@@ -106,11 +106,8 @@ COPY composer.json composer.lock ./
 COPY patches/ patches/
 
 # Copy Project
-COPY docroot/modules/custom docroot/modules/custom
-COPY docroot/themes/custom docroot/themes/custom
-COPY docroot/profiles docroot/profiles
+COPY docroot/ docroot/
```

Comment:
Needed to copy the new `health` directory over, and figured it was easier/better to copy the whole directory.  Everything that's in docroot should be made available, as Drupal expects this folder to be published.